Assets/Editor/BuildTiltBrush.cs

```diff
@@ -651,6 +664,14 @@ class TempSetCommandLineOnlyPlayerSettings : IDisposable {
         } else if (args[i] == "-btb-increment-bundle-version") {
           // Don't restore this, because user might want to check in this change?
           PlayerSettings.Android.bundleVersionCode += 1;
+        } else if (args[i] == "-buildVersion") {
+          // TODO: do we want to do anything with this? Can we use it instead of the version string set externally?
+          i++;
+        } else if (args[i] == "-androidVersionCode" || args[i] == "-androidKeystoreName" || args[i] == "-androidKeystorePass" || args[i] == "-androidKeyaliasName" || args[i] == "-androidKeyaliasPass") {
```

Comment:
nit:
On Tilt Brush we limited all code lines to 100 characters in length. I don't think we've made any decision on whether to continue with that on Open Brush, but if we want to keep consistent we should do the same.

Reply:
Sure, makes sense. I’m going to open a separate PR for automated checking this as well. No reason for a reviewer to need to look for this type of thing!